README.md

```diff
@@ -63,6 +65,8 @@ Things that explain how Solid works (or how to work with Solid).
 
 Things built on top of solid or that plug Solid into other things.
 
+- [Solid Nextcloud](https://pdsinterop.org/solid-nextcloud) - Turns a Nextcloud Server into a Solid Pod
+  ([repo](https://github.com/pdsinterop/solid-nextcloud)).
```

Comment:
Second link to the repo is maybe superfluous?

Reply:
Not sure that I agree. The main link https://pdsinterop.org/solid-nextcloud is more project-related / consumer-facing. The repo is added as a bonus. 

The setup seems to work well with the current content (see https://pdsinterop.org/awesome-solid/).